doc/making-a-visualizer.md

```diff
@@ -1,17 +1,212 @@
 ## Building a Visualizer
 
-Big picture, every visualizer needs to implement the interface defined in
-`VisualizerInterface.ts` which specifies the basic expectations of a
-visualizer. (See below for the easy way to do this.) This interface includes
-the following data and methods:
+### Get started
+
+#### Choose a graphics framework
+
+The easiest way to build a visualizer is to extend a pre-made visualizer base
+class, which automatically sets up a graphics framework for you to use. Right
+now, there's only one base class available:
+
+-   [`P5Visualizer`](#p5-visualizers) uses the
+    [**p5.js**](https://p5js.org/learn/) library for graphics and user
+    interaction.
+
+If you want to use a new graphics framework, you'll need to write your own
+implementation of the [visualizer interface](#abstract-visualizers).
+
+#### Test your visualizer on the workbench
+
+While you're working on a visualizer, we recommend keeping it in the
+`src/visualizers-workbench` directory, where Frontscope typically won't notice
+it. To load it, run Frontscope in "workbench mode" by calling
```

Comment:
I just ripgrepped and found that nowhere else do we capitalize frontscope. So please either say "Numberscope" or "frontscope" or "the front end" here, whichever you think is more appropriate in each of the instances. Thanks.

Reply:
Done.

Reply:
Found one more that will be fixed in the next commit I push, so closing this.